robert_beverage/scripts/generate_qr_code

```diff
@@ -46,10 +46,10 @@ parser.add_argument('--filename', metavar='f', type=str, default=["qr_code.png"]
 args = parser.parse_args()
 
 if not host_correct(args.host[0]):
-    rospy.logerr("host is not correct")
+    raise Exception("host is not correct")
```

Comment:
Please use simple **if-else** logic to avoid the rest of the code execution. No Exceptions are needed here.